src/auth/jwtOptions.ts

```diff
@@ -21,5 +21,5 @@ export const jwtOptions: Params = {
 	}) as GetVerificationKey,
 	issuer,
 	algorithms: ['RS256'],
-	credentialsRequired: true,
+	credentialsRequired: false,
```

Comment:
Why aren't credentials required?

Reply:
Basically this means the jwt parser will parse JWTs if they exist, but if they don't exist then it just moves forward -- now we have middleware that exclusively requires the auth session (rather than both creating AND requiring the auth session)